src/ddqa/utils/jira.py

```diff
@@ -194,6 +196,25 @@ async def update_issue_status(self, client: ResponsiveNetworkClient, issue: Jira
 
         return new_issue
 
+    def get_github_user_id_from_jira_user_id(self, jira_user_id: str) -> str:
+        for github_id, jira_id in self.config.members.items():
+            if jira_id == jira_user_id:
+                return github_id
+
+        return ''
+
+    def get_jira_user_id_from_github_user_id(self, github_user_id: str) -> str:
+        return self.config.members.get(github_user_id, '')
```

Comment:
I'm guessing returning an empty string on missing key here is *convenient*, but it carries with it an implicit meaning for `''` that all the caller chain needs to agree to. Returning `None` or raising an exception might be a bit more explicit.

Reply:
Updated!